src/validation/error.messages.ts

```diff
@@ -182,6 +182,7 @@ export enum ErrorMessages {
   SELECT_IF_STILL_MANAGING_OFFICER = "Select yes if they are still a managing officer",
   SELECT_IF_ANY_TRUSTS_INVOLVED = "Select yes if there are any trusts involved in this overseas entity",
   SELECT_DO_YOU_WANT_TO_MAKE_OE_CHANGE = "Select yes if you need to make any changes to this overseas entity",
+  SELECT_DO_YOU_WANT_TO_MAKE_CHANGES_UPDATE_STATEMENT = "Select yes if this information is correct",
```

Comment:
```suggestion
  SELECT_DO_YOU_WANT_TO_MAKE_CHANGES_UPDATE_STATEMENT = "Select yes if all the information in this update statement is correct",
```

Reply:
Did the content come from the designers? Quick check with Sheena?

Reply:
I've just taken it from the ticket, I can get this double checked though

Reply:
I didn't even notice his comment. I've changed it to match.